server/shadow/cli/shadow.c

```diff
@@ -62,6 +62,10 @@ int main(int argc, char** argv)
 		  "Clients must authenticate" },
 		{ "remote-guard", COMMAND_LINE_VALUE_BOOL, NULL, BoolValueFalse, NULL, -1, NULL,
 		  "Remote credential guard" },
+		{ "restricted-admin", COMMAND_LINE_VALUE_BOOL, NULL, BoolValueTrue, NULL, -1, NULL,
```

Comment:
might be a good idea to split this commit as `vmconnect` and `restricted-admin` are separate things?

Reply:
no, fine this way (just easier to track later on with separate commits.